.github/workflows/composite/collectlogs/action.yaml

```diff
@@ -0,0 +1,37 @@
+name: "collectlogs"
+description: "Collect and upload the logs"
+
+inputs:
+  artifact-name:
+    description: "the name of artifacts to store"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: E2E Tests
+      shell: bash
+      run: |
+        export KUBECONFIG=${HOME}/.kube/config
+        make test-validation
+        make test-e2e
```

Comment:
Better to have this step out as part of the other jobs and make this workflow to only collect logs.